.versions

```diff
@@ -14,7 +14,7 @@ NPM_VERSION=8.15.1
 NUGET_INSPECTOR_VERSION=0.9.12
 PHP_VERSION=8.1
 PIPTOOL_VERSION=22.2.2
-PNPM_VERSION=7.8.0
+PNPM_VERSION=8.8.0
```

Comment:
As this upgrade breaks the tests, I was wondering if updating lockfiles should be squashed. I'll leave it up to you. Without squashing I believe it makes sense to at least mention that tests break and lockfiles are updated in a following commit.